core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/export/localfile/LocalFileExporter.java

```diff
@@ -56,8 +57,9 @@ public Iterable<String> get(final String filePath) throws OperationException {
             } else {
                 fileToBeRead = StreamUtil.openStream(LocalFileExporter.class, filePath);
             }
-            BufferedReader reader = new BufferedReader(new InputStreamReader(fileToBeRead));
+            BufferedReader reader = new BufferedReader(new InputStreamReader(fileToBeRead, StandardCharsets.UTF_8));
             linesFromFile = reader.lines().collect(Collectors.toList());
+            reader.close();
         } catch (final IOException e) {
```

Comment:
Should this be in another `try` loop in case an error occurs before the `reader` can be closed?

Reply:
I've now changed this part to be in a nested try with resources. It will now always be closed and any exception caught by the original catch. I've also tidied the code up in this class to be more concise in general.

Reply:
It is still seems to be the same code to me... No `try-with-resources`

Reply:
Seems it has been changed after all. Just Github not showing the change. Looks good to me